test/common/3scale_smtp.go

```diff
@@ -477,6 +493,27 @@ func sendTestEmail(ctx *TestingContext, t TestingTB) {
 	}
 }
 
+func reset3ScaleSecret(ctx *TestingContext, t TestingTB) (string, error) {
+	t.Log("Resetting 3Scale secret")
+	secret := v1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "system-smtp",
+			Namespace: NamespacePrefix + "3scale",
+		},
+		Data: map[string][]byte{},
+	}
+	secret.Data["address"] = []byte(original3ScaleHost)
+	secret.Data["password"] = []byte(original3ScalePassword)
+	secret.Data["port"] = []byte(original3ScalePort)
+	secret.Data["username"] = []byte(original3ScaleUsername)
+
+	if err := ctx.Client.Update(goctx.TODO(), secret.DeepCopy(), &k8sclient.UpdateOptions{}); err != nil {
+		return secret.APIVersion, err
```

Comment:
```suggestion
		"", err
```
we don't seem to be using the secret at all, why return it at all?